Steaker/Controller/CookingViewController.swift

```diff
@@ -45,27 +45,28 @@ class CookingViewController: UIViewController{
         timerLabel.adjustsFontForContentSizeCategory = true
         turnsLeftLabel.adjustsFontForContentSizeCategory = true
         
-        updateTurnsLabel(turns: Int(highTempTurns!)!)
+        updateTurnsLabel(turns: highTempTurns!)
         
-        timerLabel.text = highTempTime!
+        timerLabel.text = String(highTempTime!)
```

Comment:
lepiej zrobić ```highTempTime?.description()``` , ponieważ text od UILabela jest optionalem i jak  ```highTempTime``` będzie nilem albo czymś innym niż string to nie będzie crasha aplikacji a jak zrobiłeś force(!), a ```highTempTime``` będzie nie stringiem to wywali aplikację całą